circleci/config.yml.tt

```diff
@@ -93,17 +100,6 @@ workflows:
   commit-workflow:
     jobs:
       - static_analysis
-      - test
-      - test_system
-  cron-workflow:
```

Comment:
Remove the cron workflow. Apps that are actively under development don't need it.